src/types/general.graphql

```diff
@@ -37,4 +38,11 @@ type Page {
 
 type MetaResults {
     total: Int
-}
\ No newline at end of file
+}
+
+input ListInput {
+    page_offset: Int
+    page_limit: Int
```

Comment:
`page_offset` and `page_limit` has `!` in the ogrnial query, `src/types/query.graphql`.

Reply:
we should make those option in the schema. our backend has sensible defaults.

Reply:
yup that's why i removed it. we shouldn't enforce it in graphql. i have some fixes coming, let me add those changes Danial mentioned above to it then push. need to sort SSO token quickly too